scripts/init.sql

```diff
@@ -25,90 +31,81 @@ CREATE TABLE "children" (
     "birth_year" integer NOT NULL,
     "interest" text NOT NULL,
     "story" text NOT NULL,
-    "image_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+    "image_id" uuid,
+    "agency_id" uuid NOT NULL,
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP)
 );
 
 CREATE TABLE "community_posts" (
     "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
     "message" text NOT NULL,
     "agency_id" uuid NOT NULL,
-    "image_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+    "image_id" uuid,
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP)
 );
 
 CREATE TABLE "images" (
     "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
     "url" varchar(255) NOT NULL,
     "meta_data" json,
     "created_by" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP)
 );
 
 CREATE TABLE "items" (
     "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
     "name" varchar(255) NOT NULL,
     "price" numeric(10, 2) NOT NULL,
     "link" varchar(2000) NOT NULL,
-    "retailer" varchar(255) NOT NULL,
+    "retailer_name" varchar(255) NOT NULL,
     "retailer_product_id" varchar(255) NOT NULL,
     "meta_data" json,
-    "image_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+    "image_id" uuid,
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP)
 );
 
 CREATE TABLE "messages" (
     "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
     "content" text NOT NULL,
     "sender_id" uuid NOT NULL,
     "wishcard_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP)
+    "created_at" timestamptz NOT NULL DEFAULT (CURRENT_TIMESTAMP)
 );
 
 CREATE TABLE "orders" (
     "id" uuid PRIMARY KEY NOT NULL DEFAULT (gen_random_uuid()),
-    "status" integer NOT NULL DEFAULT 0,
-    "delivery_date" timestamp,
+    "status" OrderStatus NOT NULL DEFAULT ('pending'::OrderStatus),
+    "delivery_date" timestamptz,
     "tracking_info" varchar(500),
     "donor_id" uuid NOT NULL,
-    "child_id" uuid NOT NULL,
-    "item_id" uuid NOT NULL,
-    "agency_id" uuid NOT NULL,
-    "created_at" timestamp NOT NULL DEFAULT (CURRENT_TIMESTAMP),
-    "updated_at" timestamp
+    "wishcard_id" uuid NOT NULL,
```

Comment:
removed all unnecessary relations to the child/agency/item when they can instead be retrieved by checking the associated tables via the `wishcard_id`.